### PR TITLE
add labels to bestfit json and lightcurve plot

### PR DESCRIPTION
**Pull request by tylerbarna**
adds label to filename of the bestfit json, also adds a gitignore so personal vscode settings are not committed to the repo

Slightly alters work from #147, related to #138

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,6 +5,9 @@ __pycache__/
 nmma.egg-info/
 .DS_Store
 
+# development
+.vscode/
+
 # models
 svdmodels/Bu2019lm_lbol.pkl
```
Comment (mcoughlin) [inline .gitignore:12]: Can we modify the ignore to ignore everything in this folder now?

**Review by mcoughlin — APPROVED** [review]
Minor comment on the ignore file.

File: .gitignore (hunk 1 above)
Comment (tylerbarna) [reply .gitignore:12]: the change to gitignore should do that now, no?

Comment (mcoughlin) [reply .gitignore:12]: Ah I meant in svdmodels. I noticed that it still calls out individual files

Comment (tylerbarna) [reply .gitignore:12]: ah okay, I'll open that as a separate pr then

Comment (mcoughlin) [reply .gitignore:12]: Cool

**Pull request closed without merging**